src/api/analytics.ts

```diff
@@ -87,6 +94,16 @@ export class Analytics {
         return this.d2Api.get<AnalyticsResponse>("/analytics", options);
     }
 
+    getEnrollmentsQuery(
+        programId: Id,
+        options: AnalyticsOptions
+    ): D2ApiResponse<AnalyticsResponse> {
```

Comment:
When definining the signature of a function, the most important is how it looks like being called: `getEnrollmentsQuery("Gs69Uw2Mom1")`. But what is the id? I'd create a GetEnrollmentsQueryOptions with AnalyticsOptions + programId. Then the call looks: `getEnrollmentsQuery({programId: "Gs69Uw2Mom1"})`.